delft/textClassification/wrapper.py

```diff
@@ -226,28 +228,31 @@ def predict(self, texts, output_format='json', use_main_thread_only=False):
             return result
 
     def eval(self, x_test, y_test, use_main_thread_only=False):
+        if self.model is None:
+            raise (OSError('Could not find a model.'))
+
+        print_parameters(self.model_config.batch_size, self.training_config.early_stop,
+                         self.training_config.learning_rate, self.training_config.max_epoch,
+                         self.model_config.maxlen, self.model_config.model_name)
+        self.model.print_summary()
+
         bert_data = False
-        if self.transformer_name != None:
+        if self.transformer_name is not None:
             bert_data = True
 
         if self.model_config.fold_number == 1:
-            if self.model != None:
-                test_generator = DataGenerator(x_test, None, batch_size=self.model_config.batch_size, 
+            test_generator = DataGenerator(x_test, None, batch_size=self.model_config.batch_size,
                     maxlen=self.model_config.maxlen, list_classes=self.model_config.list_classes, 
                     embeddings=self.embeddings, shuffle=False, bert_data=bert_data, transformer_tokenizer=self.model.transformer_tokenizer)
 
-                result = self.model.predict(test_generator, use_main_thread_only=use_main_thread_only)
-            else:
-                raise (OSError('Could not find a model.'))
```

Comment:
Any reason to have removed this conditional? It was to inform the user in case he was launching an eval without a model or n models saved (or if he makes a mismatched between simple model and n models).

Reply:
I think the check for the model was moved at the beginning of the method to simplify the ifs

Reply:
Only the check on model has been moved, not the check on models (case n fold > 1).

Reply:
ahh true... let me try to fix it